examples/periodic_snapshotter.cpp

```diff
@@ -33,15 +33,25 @@
 *********************************************************************/
 
 #include <cstdio>
-#include <ros/ros.h>
-
+#include "rclcpp/rclcpp.hpp"
+#include <string>
+#include <chrono>
 // Services
-#include "laser_assembler/AssembleScans.h"
+#include "laser_assembler_srv_gen/srv/assemble_scans.hpp"
 
 // Messages
-#include "sensor_msgs/PointCloud.h"
-
-
+#include "sensor_msgs/msg/point_cloud.hpp"
+#include "rcutils/cmdline_parser.h"
+#include "rclcpp/clock.hpp"
+#include "std_msgs/msg/string.hpp"
+
+#define ROS_ERROR printf
```

Comment:
Does ROS2 not have any logging functions yet?

Reply:
ROS2 has logging functions, fixed it in the next commits.